style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```

Comment:
- [ ] Kindly note that you are required to use a CSS class selector to style the ```<h1>``` element. This will involve you adding a class to your ```<h1>```  and targeting it here to style it.

Reply:
Noted. I have modified the HTML and CSS file to use class selector to style the <h1> element.